src/crystal/scheduler.cr

```diff
@@ -123,4 +138,84 @@ class Crystal::Scheduler
     @current.resume_event.add(0.seconds)
     resume(fiber)
   end
+
+  {% if flag?(:preview_mt) %}
+    @rr_target = 0
+
+    protected def find_target_thread
+      if @@workers.empty?
+        Thread.current
+      else
+        @rr_target += 1
+        @@workers[@rr_target % @@workers.size]
+      end
+
+      # target = Thread.workers[@rr_target]
```

Comment:
Will this be removed before merge?

Reply:
Actually that's a leftover from an attempt to select the least busy thread for each new fiber. But that was buggy and unreliable. I'll remove all that and revisit in future.